integration/test/releasecycle/releasecycle_test.go

```diff
@@ -29,7 +29,7 @@ var releaseCycleCR = &releasev1alpha1.ReleaseCycle{
 var expectedAppCR = &applicationv1alpha1.App{
 	ObjectMeta: metav1.ObjectMeta{
 		Labels: map[string]string{
-			"app-operator.giantswarm.io/version": "1.0.0-" + env.CircleSHA(),
+			"app-operator.giantswarm.io/version": "0.0.0-" + env.CircleSHA(),
```

Comment:
If I'm not mistaken, this is due to change of default architect version from 1.0.0-SHA to 0.0.0-SHA via https://github.com/giantswarm/giantswarm/issues/7044

Reply:
Yes, good catch.